apps/web/src/components/content.tsx

```diff
@@ -25,6 +25,21 @@ function resolveApiUrl(): string {
   return `https://api.${env.NEXT_PUBLIC_NETWORK_NAME}.blobscan.com`;
 }
 
+type Network = typeof env.NEXT_PUBLIC_NETWORK_NAME;
+
+const NETWORKS_FIRST_BLOB_NUMBER: Record<Network, number> = {
```

Comment:
Maybe we can have something like **getNetworkDencunForkSlot** but for blocks?

https://github.com/Blobscan/blobscan/blob/03fb6b0d3291a85e80cbdab6cb497b782b17e7e8/apps/rest-api-server/src/utils.ts#L3-L20